Detector/DetCRD/compact/CRD_common_v01/DC_Simple_v01_01.xml

```diff
@@ -47,6 +47,8 @@
     <constant name="DC_inner_chamber_enabled" value="1"/>
     <constant name="DC_outer_chamber_enabled" value="1"/>
 
+    <constant name="Safe_distance" value="0.02*mm"/>
```

Comment:
Could you add the prefix, such as `DC_safe_distance`?

Reply:
OK.